frontend/packages/noobaa-storage-plugin/integration-tests/utils/consts.ts

```diff
@@ -2,6 +2,8 @@ export const OBC_NAME = 'test-obc';
 export const BOUND = 'Bound';
 export const ATTACH_TO_DEPLOYMENT = 'Attach to Deployment';
 export const OBC_RESOURCE_PATH = 'objectbucket.io~v1alpha1~ObjectBucketClaim';
+export const MINUTE = 60 * 1000;
+export const SECOND = 1000;
```

Comment:
export const SECOND = 1000;
export const MINUTE = SECOND * 60;

Reply:
I did it the other way around as it guarantees that both of them are defined under strict mode conditions. For eg., after a rebase, the order could change which would lead `MINUTE` to be undefined. WDYT?